### PR TITLE
docs: add instructions for installation using BeautifulDiscord

### PR DESCRIPTION
**Pull request by LordMZTE**
This PR adds a section to the README explaining how to install the theme using [BeautifulDiscord](https://github.com/leovoel/BeautifulDiscord).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,27 @@
...
 ## Usage
 
+### [BeautifulDiscord](https://github.com/leovoel/BeautifulDiscord)
+
+1. Download your preferred flavour:
+
+- 🌻 [Latte](https://catppuccin.github.io/discord/dist/catppuccin-latte.theme.css)
+- 🪴 [Frappe](https://catppuccin.github.io/discord/dist/catppuccin-frappe.theme.css)
+- 🌺 [Macchiato](https://catppuccin.github.io/discord/dist/catppuccin-macchiato.theme.css)
+- 🌿 [Mocha](https://catppuccin.github.io/discord/dist/catppuccin-mocha.theme.css)
```
Comment (nullishamy) [inline README.md:44]:
```suggestion
- 🌻 [Latte](./themes/latte.theme.css?raw=1)
- 🪴 [Frappe](./themes/frappe.theme.css?raw=1)
- 🌺 [Macchiato](./themes/macchiato.theme.css?raw=1)
- 🌿 [Mocha](./themes/mocha.theme.css?raw=1)
```

File: README.md [hunk 2]
```diff
@@ -34,6 +34,27 @@
...
+
+2. Move the downloaded file into a location of your choosing, for example `~/.config/discord`.
```
Comment (nullishamy) [inline README.md:46]: Windows instructions?

File: README.md [hunk 3]
```diff
@@ -34,6 +34,27 @@
...
+
+3. Ensure Discord is **running**.
+4. Install and run [BeautifulDiscord](https://github.com/leovoel/BeautifulDiscord):
```
Comment (nullishamy) [inline README.md:49]:
```suggestion
4. Install and run [BeautifulDiscord](https://github.com/leovoel/BeautifulDiscord#installing):
```

**Review by nullishamy — APPROVED** [review]

**Comment by ToxicAven** [comment]
Thanks!

**Pull request merged**